### DISCUSSION TITLE
Allow the community to create and share prisma schemas using the current playground

### DISCUSSION THREAD

**Discussion by zomars** [prisma/prisma#23954] [Q&A]
### Question

While the current playground is nice to test some queries. It would be even better being able to come up with different schemas for certain use cases and being able to share them around.

Let's say I'm comming up with some kind of authentication schema, I can create a new schema and share it via URL like:

https://playground.prisma.io/s/UUID

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hi @zomars 👋 

Thank you for raising this.

I have shared your feature request for the Prisma playground with the team.

Do let us know if you have any other questions.

**Reply by ludralph** [reply]
Alternatively, you can use [codesandbox](https://codesandbox.io/p/devbox/prisma-quickstart-58fs6w) to share the Schema.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
